### DISCUSSION TITLE
Welcome to bowser Discussions!

### DISCUSSION THREAD

**Discussion by lancedikson** [lancedikson/bowser#460] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Tanveer-Singh** [comment]
How can I use it in Javascript? with my.net core application

**Comment by NansySalem** [comment]
As you may know, starting with iPadOS 13, the Safari web browser on iPadOS defaults to sending a User-Agent string that appears to be from a macOS device instead of an iOS or an iPadOS device. Is there a way that Bowser can detect whether the device is actually an iPad? Can you give an example about the best way to do this? Thanks!

**Reply by simonIrengardIkea** [reply]
any progress on this?

**Reply by simonIrengardIkea** [reply]
Solved it like this if anyone is interested 👍  
```
  const bowserParser = Bowser.getParser(window.navigator.userAgent || window.navigator.vendor);
  const isMacOS = /macos/i.test(bowserParser.getOSName());
  const isTouchScreen = navigator.maxTouchPoints >= 1;

  // checks if it is "macOS" (ipadOS big screens) and if its possible to use touch interactions (can't do that on mac)
  const isIpadOS = isTouchScreen && isMacOS;
```
